dotcom-rendering/src/web/server/index.ts

```diff
@@ -184,6 +187,36 @@ export const renderKeyEvents = (
 	}
 };
 
+/**
+ * Sends markup to Frontend, which forwards it to the client.
+ * Frontend expects that it will receive the markup string as a value of the
+ * key 'html' in a json object, i.e. { "html": "<some markup>" }
+ */
+export const renderCards = (
+	{ body }: { body: CardsRequest },
+	res: express.Response,
+): void => {
+	try {
+		const { cards, startIndex, config } = body;
```

Comment:
Are we only using `startIndex` for tracking? This feels like a pretty big api change just for this very small edge case. Maybe we can replace the numeric index value here with the text show-more'.This would simplify this code, prevent a need for a major api edit and arguably be more useful for anyone analysing the data

Reply:
I believe there was some discussion about this before? I agree that it doesn't seem that important in itself, but quite a lot of work was going into setting this in the original Frontend endpoint, and from what I remember of previous discussions it was decided that this is something we should support..

Re: adding 'show-more' to the `data-link-name` attr, I've already done this with an optional prop on the `ExtraCardsContainer.tsx` (ln.40). Does that look like a reasonable pattern to youo?

Reply:
Maybe we can revisit the motivations around the earlier discussion? It could be there are strong reasons for having this prop that we should learn about here. At the very least we should document them for future engineers landing here and trying to understand our thinking.

My gut feeling is this prop might not be worth it but lets lay both sides out and see where we land!

Reply:
happy to discuss this, but as it involves a change to the functionality, as opposed to straightforward migration, I suggest that we discuss this as a separate issue at a later time?